src/humanloop/decorators/flow.py

```diff
@@ -0,0 +1,100 @@
+import uuid
+from functools import wraps
+from typing import Any, Callable, Optional
+
+from humanloop.decorators.helpers import args_to_inputs
+from humanloop.otel import get_humanloop_sdk_tracer, get_trace_parent_metadata, pop_trace_context, push_trace_context
+from humanloop.otel.constants import HL_FILE_OT_KEY, HL_LOG_OT_KEY, HL_OT_EMPTY_VALUE, HL_TRACE_METADATA_KEY
+from humanloop.otel.helpers import write_to_opentelemetry_span
+
+
+def flow(
```

Comment:
How is your flow decorator inferring which inputs to pass in?

And presumably it assumes a string response?

We need to think about how this is communicated to the user. Assuming they will need to refactor their code to use this?

Reply:
Inputs are taken from the arguments the decorated function and the output is the return value of the decorated function.

There is a standing problem because the inputs and output are expected to be strings by HL API. I will look into this

Reply:
Just confirming this was added?

Reply:
ack - output is jsonified or an error is thrown. I've updated the docstrings on the client